doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tls-support-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='kie-server-smart-router-enable-tls-support-proc_{context}']
 = Configuring Smart Router for TLS support
 
-You can configure Smart Router (KIE Server Router) for TLS support to allow HTTPS traffic.
+You can configure Smart Router (KIE Server Router) for Transport Layer Security (TLS) support to allow HTTPS traffic. In addition, you can disable non-secure HTTP connections to Smart Router.
```

Comment:
[style] Would "unsecured" be better in place of "non-secured"? Maybe rephrase for translatability?

s/In addition, you can disable non-secure HTTP connections to Smart Router/In addition, you can disable unsecured HTTP connections to Smart Router./In addition, you can disable HTTP connections to Smart Router that would not be secure.

Maybe rephrase in terms of the mechanism of Smart Router blocking or rejecting HTTP connections?

"You can also configure Smart Router to reject unsecured HTTP connection attempts."